src/enkf/observer_fv3reg.f90

```diff
@@ -62,7 +62,8 @@ subroutine setup_linhx(rlat, rlon, time, ix, delx, ixp, delxp, iy, dely,  &
 
 end subroutine setup_linhx
 
-subroutine calc_linhx(hx, dens, dhx_dx, hx_ens, &
```

Comment:
Ting, do those changes come from you?

Reply:
Yes, I think this is to make it has the same form as the definition in observer_gfs.f90  and the one  called in the readconvobs.f90 and son on.